File: example/Simplace_Soil_Temperature/src/cpp/Simplace_Soil_Temperature/SoilTemperatureAuxiliary.cpp
#include "SoilTemperatureAuxiliary.h"
using namespace Simplace_Soil_Temperature;

SoilTemperatureAuxiliary::SoilTemperatureAuxiliary() {}

double SoilTemperatureAuxiliary::getSnowIsolationIndex() { return this->SnowIsolationIndex; }

void SoilTemperatureAuxiliary::setSnowIsolationIndex(double _SnowIsolationIndex) { this->SnowIsolationIndex = _SnowIsolationIndex; }

File: example/Simplace_Soil_Temperature/src/java/Simplace_Soil_Temperature/SoilTemperatureAuxiliary.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class SoilTemperatureAuxiliary
{
    private double SnowIsolationIndex;
    
    public SoilTemperatureAuxiliary() { }
    
    public SoilTemperatureAuxiliary(SoilTemperatureAuxiliary toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.SnowIsolationIndex = toCopy.getSnowIsolationIndex();
        }
    }
    public double getSnowIsolationIndex()
    { return SnowIsolationIndex; }

    public void setSnowIsolationIndex(double _SnowIsolationIndex)
    { this.SnowIsolationIndex= _SnowIsolationIndex; } 
    
}